### PR TITLE
6.0.8 release notes

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/release_notes_128t_6.0.md [hunk 1]
```diff
@@ -28,6 +28,115 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-49118 HA LTE Interfaces go down and impact BGPoSVR and Conductor:** The handling of FIB updates due to interface state changes has been optimized to avoid possible traffic loss for unaffected FIB entries.
+------
+- **I95-49134 DHCP server does not work when device IDs on HA interface do not match:** Resolved an issue where a DHCP server interface may instead forward DHCP requests through the `service-area` and out to the WAN.
+------
+- **I95-49166 OSPF is not configurable using the GUI:** This issue has been resolved.
+------
+- **I95-49225 Packets containing only path-metrics metadata are dropped:** Resolved an issue where FPM calculations caused these packets to be dropped when flows were affected due to routing changes.
+------
+- **I95-49242 When HMAC is disabled, the automatic MSS adjustment calculation for `enforced-mss = automatic` may be wrong :** The Automatic MSS adjustment calculation has been corrected (expanded).
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:120]: space after wrong and :

File: docs/release_notes_128t_6.0.md [hunk 2]
```diff
@@ -28,6 +28,115 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-49225 Packets containing only path-metrics metadata are dropped:** Resolved an issue where FPM calculations caused these packets to be dropped when flows were affected due to routing changes.
+------
+- **I95-49242 When HMAC is disabled, the automatic MSS adjustment calculation for `enforced-mss = automatic` may be wrong :** The Automatic MSS adjustment calculation has been corrected (expanded). 
+------
+- **I95-49341 BGP next hop exception being thrown:** Resolved an issue where a duplicate BGP next hop resulted in an exception.
+------
+- **I95-49451 Secondary conductor failed to synchronize after upgrade:** This issue was resolved by re-ordering the start up process for the 128T service.
+------
+- **I95-49454 Error while creating a new Radius user from the GUI:** The create user API will now reject requests that specifies invalid input parameters.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:126]: "specifies invalid input" - not sure what this means.
should it instead be "... will now reject requests that have invalid input parameters"?

File: docs/release_notes_128t_6.0.md [hunk 3]
```diff
@@ -28,6 +28,115 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-49242 When HMAC is disabled, the automatic MSS adjustment calculation for `enforced-mss = automatic` may be wrong :** The Automatic MSS adjustment calculation has been corrected (expanded). 
+------
+- **I95-49341 BGP next hop exception being thrown:** Resolved an issue where a duplicate BGP next hop resulted in an exception.
+------
+- **I95-49451 Secondary conductor failed to synchronize after upgrade:** This issue was resolved by re-ordering the start up process for the 128T service.
+------
+- **I95-49454 Error while creating a new Radius user from the GUI:** The create user API will now reject requests that specifies invalid input parameters. 
+------
+- **I95-49455 Cut and paste from `show flat` generates an error:** Resolved the issue that prevented editing the OSPF list.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:128]: incorrect Jira id. https://128technology.atlassian.net/browse/I95-49455 is for RTT stats.
correct jira id is https://128technology.atlassian.net/browse/I95-49655
Also, I believe the wording of this to be incorrect. 
cutting and subsequently pasting the output of `show flat` does not work for OSPF

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_6.0.md [hunk 4]
```diff
@@ -28,6 +28,115 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-48641 Recreating BFD flow when an outbound-only session is reset:** Flow creation is now deferred until a reverse packet arrives from the peer, similar to the initial creation case.
+------
+- **I95-48684 SSR not answering ARP requests:** Increased `internal-application traffic-engineering` rates for ARP traffic which was being dropped in a multiple packet-processing core environment incorrectly due to an over aggressive traffic engineering profile.
+------
+- **I95-48689 Top Sessions not displaying source address:** Restored the **Source** column in the Top Sessions table. 
+------
+- **I95-48723 HA sync not running after systems reconnect:** Historical metrics and events are synced between HA nodes after extended downtime.
+------
+- **I95-48869 Failed to identify default IDP policy when it appears to be the first access policy defined in a service:** The default IDP policy is now handled as `IDP off` in all cases.
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_6.0.md]: I had gotten this info from Sunan, but I like your wording better.

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**